ui/cryptomaterial/wallet_clickable_list.go

```diff
@@ -0,0 +1,116 @@
+package cryptomaterial
+
+import (
+	"gioui.org/layout"
+	"gioui.org/unit"
+	"github.com/crypto-power/cryptopower/libwallet/utils"
+)
+
+type SelectedWalletItem struct {
+	AssetType utils.AssetType
+	Index     int
+}
+
+type WalletClickableList struct {
+	layout.List
+	theme           *Theme
+	clickables      []*Clickable
+	Radius          CornerRadius // this radius is used by the clickable
+	selectedItem    int
+	DividerHeight   unit.Dp
+	IsShadowEnabled bool
+	IsHoverable     bool
+	ItemIDs         map[*Clickable]SelectedWalletItem
+}
+
+func (t *Theme) NewWalletClickableList(axis layout.Axis) *WalletClickableList {
+	click := &WalletClickableList{
+		theme:        t,
+		selectedItem: -1,
+		List: layout.List{
+			Axis: axis,
+		},
+		IsHoverable: true,
+	}
+
+	return click
+}
+
+func (cl *WalletClickableList) ItemClicked() (bool, SelectedWalletItem) {
+	for _, clickable := range cl.clickables {
+		if clickable.Clicked() {
+			if itemID, exists := cl.ItemIDs[clickable]; exists {
+				return true, itemID
```

Comment:
We already have this in the wallet_selector_page. No need doing this here again.